addons/website/models/theme_models.py

```diff
@@ -192,6 +192,30 @@ class Theme(models.AbstractModel):
     _description = 'Theme Utils'
     _auto = False
 
+    _header_templates = [
```

Comment:
Should not this commit be at least for 14.0? (when we introduced those templates (maybe different ones though) I think ?)

Reply:
It doesn't produce any blocking issue in 14.0, that's why I targeted 15.0, but I can if you are down for it.

Reply:
Maybe not "blocking" but still buggy to me. You can have two header templates enabled at the same time **and maybe not the one that was enabled last** as it is just the view sequence order that determines the final header.

Reply:
As discussed, we will backport this afterward, as all PRs are ready and linked everywhere. That would be more easy to understand and convenient to keep this commit inside this PR.
Plus we will let it live a few days in 15.0 before backporting in 14.0, better safe than sorry.